src/select/typeahead/TypeaheadSelect.tsx

```diff
@@ -206,6 +208,15 @@ function TypeaheadSelect({
       setKeyword(value);
     }
   }
+
+  function handleKeyDown(event: React.KeyboardEvent<HTMLInputElement>) {
+    const {key} = event;
+
+    if (key === KEYBOARD_EVENT_KEY.BACKSPACE && !inputValue && onTagRemove) {
```

Comment:
I think we should also check if there are any selectedOptions available to remove